packages/ngx-flicking/projects/ngx-flicking/package.json

```diff
@@ -1,12 +1,28 @@
 {
-  "name": "ngx-flicking",
-  "version": "0.1.0",
+  "name": "@egjs/ngx-flicking",
+  "version": "3.0.0-beta",
+  "description": "A module used to implement flicking interactions. With this module, you can make flicking gestures, which are ways to navigate left and right to move between panels arranged side by side.",
   "peerDependencies": {
     "@angular/common": "^7.2.0",
-    "@angular/core": "^7.2.0"
+    "@angular/core": "^7.2.15"
   },
   "dependencies": {
     "@egjs/flicking": "^3.1.1",
     "@egjs/ngx-children-differ": "^0.1.0"
-  }
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/naver/egjs-flicking"
```

Comment:
I think we should use angular version README file instead.
That is, https://github.com/naver/egjs-flicking/tree/master/packages/ngx-flicking